src/pages/home/report/ReportTypingIndicator.js

```diff
@@ -26,66 +26,52 @@ const defaultProps = {
     userTypingStatuses: {},
 };
 
-class ReportTypingIndicator extends React.Component {
-    constructor(props) {
-        super(props);
+const ReportTypingIndicator = (props) => {
```

Comment:
[STYLE.md](https://expensify.slack.com/archives/C01GTK53T8Q/p1686062674061049?thread_ts=1685727880.152119&cid=C01GTK53T8Q)
> Should I declare my components with arrow functions (const) or the function keyword?
There are pros and cons of each, but ultimately we have standardized on using the function keyword to align things more with modern React conventions. 

There are also a lot of great tips on migration to functional components in [this](https://expensify.slack.com/archives/C01GTK53T8Q/p1686062674061049?thread_ts=1685727880.152119&cid=C01GTK53T8Q) thread on slack

Reply:
Thanks for the link, I was in doubt between function or const, and I think we are using function

Reply:
like this one?

`prefer-arrow-callback": ["error", { "allowNamedFunctions": true }]`

Reply:
like the following

`func-style: ["error", "declaration", { "allowArrowFunctions": true }]`

Reply:
@hayata-suenaga, we don't have a lint setup for this. We also have a lot of components that are defined with arrow functions (before this was decided I guess)

Reply:
ah I see I gonna check on this with the internal team. thanks for checking on this 🙇